pkg/netutil/netutil_test.go

```diff
@@ -97,3 +128,136 @@ func TestParseIPAMRange(t *testing.T) {
 		}
 	}
 }
+
+// Tests whether nerdctl properly creates the default network when required.
+func TestDefaultNetworkCreation(t *testing.T) {
+	testutil.RequiresRoot(t)
```

Comment:
not sure if this check (RequiresRoot) is within nerdctl responsibilities  ( cc @AkihiroSuda )

Reply:
Added, though worth mentioning that `rootlessutil` isn't windows-compatible yet so this test will incorrectly be skipped. (will leave any updates to `rootlessutil` for a separate PR)